styles/modules/modals/_connectionManagement.scss

```diff
@@ -0,0 +1,94 @@
+.connectionManagement {
+  .configurations {
+    position: relative;
+
+    .statusBarOuterWrap {
+      position: absolute;
+      top: 0;
+      left: 0;
+      right: 115px;
+      height: 58px;
+
+      .statusBarMessage {
+        height: 100%;
+
+        .btnClose {
+          position: absolute;
+          right: 6px;
+          top: -4px;
+
+          span {
+            font-size: 2.8rem;
+          }
+        }
+
+        .connectFailed {
+          border-left-width: 4px;
+          border-left-style: solid;
+        }
+
+        .connectingMsg {
+          padding-left: 25px;
+
+          .spinner {
+            transform: translateX(-25px);
+          }
+        }
+      }
+    }
+
+    // this was the least ugly way I found to do this :(
+    .borderFillIn {
+      position: absolute;
+      right: -115px;
+      top: 58px;
+      width: 150px;
+    }
+
+    .configuration {
+      position: relative;
+      height: 37px;
+
+      .btnConnectCancel {
+        width: 140px;
+        padding-left: 0;
+        padding-right: 0;
+      }
+
+      .connectedText {
+        position: relative;
+
+        span {
+          font-size: $tx1;
+          // Icon is getting an abnormally large height, which messes up the vertical centering
+          // of the text. Therefore, we'll absolutely position.
```

Comment:
Well, I actually had to add `flexVCent flexHRight gutterHTn` to the parent, a `tx1` to the icon span and then wrap the "Connected" text in a span so the gutterHTn works.